diff --git a/pom.xml b/pom.xml
index 3538a84..cc2e4ad 100644
--- a/pom.xml
+++ b/pom.xml
@@ -70,7 +70,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>21</java.version>
 
-    <ver.jena>5.6.0</ver.jena>
+    <ver.jena>6.0.0-SNAPSHOT</ver.jena>
     <ver.jetty>12.1.3</ver.jetty>
 
     <!-- Ensure we get a version compatible with Jena. -->
diff --git a/rdf-delta-server-http/src/main/java/org/seaborne/delta/server/http/PatchLogServer.java b/rdf-delta-server-http/src/main/java/org/seaborne/delta/server/http/PatchLogServer.java
index 1c28b71..692fdcb 100644
--- a/rdf-delta-server-http/src/main/java/org/seaborne/delta/server/http/PatchLogServer.java
+++ b/rdf-delta-server-http/src/main/java/org/seaborne/delta/server/http/PatchLogServer.java
@@ -29,9 +29,9 @@ import jakarta.servlet.http.HttpServletResponse;
 import org.apache.jena.fuseki.main.sys.FusekiErrorHandler;
 import org.apache.jena.fuseki.servlets.ServletOps;
 import org.apache.jena.riot.Lang;
-import org.eclipse.jetty.ee10.servlet.FilterHolder;
-import org.eclipse.jetty.ee10.servlet.ServletContextHandler;
-import org.eclipse.jetty.ee10.servlet.ServletHolder;
+import org.eclipse.jetty.ee11.servlet.FilterHolder;
+import org.eclipse.jetty.ee11.servlet.ServletContextHandler;
+import org.eclipse.jetty.ee11.servlet.ServletHolder;
 import org.eclipse.jetty.http.MimeTypes;
 import org.eclipse.jetty.server.HttpConnectionFactory;
 import org.eclipse.jetty.server.Server;
